autoreject/utils.py

```diff
@@ -43,21 +43,22 @@ def _check_data(epochs, picks, ch_constraint='data_channels',
         mne.io.meas_info.channel_type(picked_info, idx)
         for idx in range(len(picks))}
 
-    if not _check_ch_locs(picked_info['chs']):
+    if check_loc and not _check_ch_locs(picked_info['chs']):
         raise RuntimeError('Valid channel positions are needed '
                            'for autoreject to work')
 
     # XXX : ch_constraint -> allow_many_types=True | False
     if ch_constraint == 'data_channels':
-        if not all(ch in ('mag', 'grad', 'eeg', 'hbo', 'hbr')
+        if not all(ch in ('mag', 'grad', 'eeg', 'hbo', 'hbr', 'ecog', 'seeg')
```

Comment:
didn't you add these in the last PR?

Reply:
`_check_data` is not used in `get_rejection_threshold`, so this utils.py file was not changed before